src/Backend/Request.php

```diff
@@ -25,21 +24,6 @@ public function __construct(ClientInterface $client)
     }
 
     /**
-     * @param string $url
-     * @param string $method
-     * @param array  $options
-     *
-     * @return RequestInterface
-     */
-    protected function createRequest($url, $method = 'GET', $options = [])
```

Comment:
According to http://semver.org/, every modification or removing of existing methods is a BC break.

I didn't mean that you should make the method private NOW, but in when creating a NEW method you should think about it. If the method is api (public or protected) or internal (private).

We should add the method for the 5.x release because this can break some projects.